src/Raven.Server/Documents/PeriodicBackup/BackupTask.cs

```diff
@@ -866,8 +867,23 @@ public static void SaveBackupStatus(PeriodicBackupStatus status, DocumentDatabas
                     PeriodicBackupStatus = status
                 };
 
-                var result = AsyncHelpers.RunSync(() => documentDatabase.ServerStore.SendToLeaderAsync(command));
-                AsyncHelpers.RunSync(() => documentDatabase.ServerStore.WaitForCommitIndexChange(RachisConsensus.CommitIndexModification.GreaterOrEqual, result.Index));
+                var index = await BackupHelper.RunWithRetriesAsync(maxRetries: 10, () =>
+                    {
+                        var result = AsyncHelpers.RunSync(() => documentDatabase.ServerStore.SendToLeaderAsync(command));
+                        return Task.FromResult(result.Index);
+                    },
+                    infoMessage: "Saving the backup status in cluster",
+                    errorMessage: "Failed to save the backup status in cluster",
+                    backupResult);
```

Comment:
Need to pass the cancellation token here

Reply:
Done.